sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,196 @@
 import pytest
 from azure.eventhub.extensions.checkpointstoretable import TableCheckpointStore
+from azure.data.tables import TableServiceClient
+import uuid
+import warnings
+import random
+import os
+from azure.eventhub.exceptions import OwnershipLostError
 
+STORAGE_CONN_STR = [os.environ.get("AZURE_STORAGE_CONN_STR", "Azure Storage Connection String")]
 
-def test_constructor():
-    client = TableCheckpointStore()
-    assert client is not None
-    
+def get_live_storage_table_client(storage_connection_str):
+    try:
+        table_name = chr(random.randint(ord('a'), ord('z'))) +(uuid.uuid4().hex)
```

Comment:
hmm, so I think if you're trying to make sure that the table name is unique and doesn't result in any errors with the Table already exisiting, I think you're fine just sticking with uuid.uuid4() to make it unique. I don't think the rest should really make a difference.
```suggestion
        table_name = `table_{}`.format(uuid.uuid4())
```

Reply:
I will make the change. the only thing I will remove is the underscore and add .hex since table name is alphanumeric